src/App.css

```diff
@@ -145,3 +145,45 @@ nav {
 a.active {
   text-decoration: underline;
 }
+
+.spinner-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  max-width: 65rem;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0;
+}
+
+.spinner {
+  position: absolute;
+  display: inline-block;
+  width: 30px;
+  height: 30px;
+  margin: 10px;
+  border: 4px solid rgba(0, 0, 0, 0.1);
+  border-left-color: #b4cddd;
+  border-radius: 50%;
+  animation: donut-spin 1.2s linear infinite;
+}
+
+@keyframes donut-spin {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.search__error h2 {
+  text-align: center;
+}
+
+.gm-style img {
```

Comment:
I saw some differences with Google Maps' buttons.

Now:

![image](https://user-images.githubusercontent.com/8689897/126253591-ff1b4ef6-6c1e-4dda-9008-686a1abc287c.png)

Usually:

![image](https://user-images.githubusercontent.com/8689897/126253794-45a97d22-afa1-4e31-a351-01c63b32fa78.png)

Reply:
Thank you for pointing this out, we noticed the bug too.  I think the `gm-style img` is applying to every image the map generates which is affecting the control icons too since they're images nested in a button element.  We're working on trying to fix this now.

Reply:
@meganmckissack found a way to that😄.